page/about.html

```diff
@@ -22,7 +22,7 @@
         <div class="container">
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done I really appreciate what you have done so far but `main texts` need to correct style, please style them like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

- `[HINT]` The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

  * Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`

- The pages use the same fonts or simialr fonts from the original design.

  * Fonts used originally: `Cocogoose, Lato`

- All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

- `[MOBILE]`
![image](https://github.com/ahmidou09/CreativeStudioCapstone/assets/63915024/813172ca-e24a-418e-b7b6-765fc1175ae9)